docs/core-tech/transaction.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 6
+title: Transaction
+sidebar_label: Transaction
+---
+
+A transaction refers to the act of transferring an entity from one address to another within the network. It is a fundamental operation that enables the exchange of Alephium's native cryptocurrency ALPH, or any other tokens or contract data.
+
+## Prerequisites
+
+It's important that you understand the [UTXO model](/core-tech/stateful-utxo)
+
+## Overview of a transaction
+
+A transaction typically includes the following components:
+
+* Inputs: list of previous unspent outputs, transaction's signatures will prove that those inputs can be spent.
+* Outputs: list of recipients address with given amount of values (such as ALPH)
+* Transaction Fee: A small amount of cryptocurrency paid by the sender to to get his transaction included in a block.
+
+When a transaction is created, it undergoes a validation process to ensure that the inputs have sufficient funds, the transaction is properly signed, UTXO model is respected and other relevant conditions are met. 
+Once validated, the transaction is added to a [block](/core-tech/block) and subsequently added to the [BlockFlow](/core-tech/blockflow-and-sharding), becoming a permanent part of Alephium's history.
```

Comment:
Q: do we say a block is added to blockflow, since blockflow is an algorithm?

Reply:
should we say something like: and subsequently added to one of our sharded blockchain?

Reply:
The blockchain in which the bloc is added is not sharded (it's a single chain) so maybe "subsequently added to one of the blockchain of the Alephium's network". You could also add at the end "according to the groups of the addresses involved in the transaction".